lib/cldr-plurals/javascript_emitter.rb

```diff
@@ -84,6 +82,11 @@ def emit_value_set(value_set, operand, operator)
         end
       end
 
+      def emit_range_check(range, operand)
+        # a..b represents all *integers* between a and b, inclusive.
+        (range.start..range.finish).to_a.map { |v| "(#{emit(operand)} == #{v})" }.join(" || ")
```

Comment:
Yes, I believe === makes sense here, my JS is a little rusty and I forgot that's the preferred equality. As long as 1 === 1.000  is true, which it is since JS represents all numbers a doubles, we should be good.  That said, this still gets ugly in the 11..99 cases. I was focusing on correctness vs optimizations at the moment.